### DISCUSSION TITLE
Request for Update on æternity Blockchain Snap Integration with MetaMask

### DISCUSSION THREAD

**Discussion by donnagifts** [MetaMask/snaps#2437] [Show and tell]
Hello Team,

I am reaching out to follow up on our prior communications regarding the integration of our æternity Blockchain Snap with MetaMask. To date, we have not received any updates or feedback on our submission, which was sent on April 8.

This integration is a priority for us, and we are committed to meeting all necessary requirements and standards to ensure a successful collaboration. Multiple emails have been sent to Manish Tomer and Chaima Khiari without any response.

Could you please provide an update on the status of our application? If there are any additional steps or documentation required from our side, kindly let us know at your earliest convenience. We are eager to proceed and are prepared to take any actions needed to facilitate this process.

Your prompt attention to this matter would be greatly appreciated.

Thank you.

Best regards,  
Donna
Donna@aeternity.foundation
Telegram:@Donnagift

**Comment by Montoya** [comment]
Hi, where have you applied? I am not seeing your submission. Did you fill out this form? https://consensys.notion.site/Process-for-publishing-the-Snap-fc33d2ef9f0e4f95945563b990e2bd7d

**Reply by donnagifts** [reply]
I wanted to inform you that I initially filled out the form on April 6th, 2024, and have recently submitted it again.

Please let me know if you require any further information or if there are additional steps I need to take.

Thank you for your attention to this matter.

**Comment by donnagifts** [comment]
Hey Christian

I wanted to inform you that I initially filled out the form on April 6th,
2024, and have recently submitted it again.

Please let me know if you require any further information or if there are
additional steps I need to take.

Thank you for your attention to this matter.

regards

Le mar. 28 mai 2024 à 14:09, Christian Montoya ***@***.***> a
écrit :

**Comment by donnagifts** [comment]
Hey Christian,

Just a quick follow-up regarding the form I initially filled out on April 6th, 2024, and resubmitted recently. Please let me know if any further information is needed or if there are additional steps I should take.

Thank you for your attention.

Regards,
Donna
